src/api/seeders/20190703095400-reasons.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('Reasons',
+    [
+      {
+        name: 'VIOLENT',
```

Comment:
@MCFrank16 We probably want this `name` attribute to be something like `reason` or `description`. And it should be a descriptive text like `it contains violent content`.

Reply:
okay, lemme change it